src/views/common/TimeScrollView/TimeScrollViewInteractions/divRefHandling.ts

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
just providing some more semantic function names and some consistency for a few bits of code that got repeated a lot.